File: backend/HRManagementSystem/src/Core/HRManagementSystem.Domain/Entities/Feedback.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using HRManagementSystem.Domain.Enums.Performance;

namespace HRManagementSystem.Domain.Entities;
public class Feedback : BaseEntity
{
    public int PerformanceReviewId { get; set; }
    public int FromEmployeeId { get; set; } 
    public FeedbackType Type { get; set; }
    public string Comments { get; set; } = default!;
    public DateTime SubmittedAt { get; set; } = DateTime.UtcNow;

    public PerformanceReview Review { get; set; } = default!;
}

File: frontend/src/app/models/performance/feedback.ts
import { FeedbackType } from './enums';

export interface Feedback {
  id: number;
  performanceReviewId: number;
  fromEmployeeId: number;
  type: FeedbackType;
  comments: string;
  submittedAt: string;
  fromEmployeeName?: string;
}

export interface CreateFeedbackDto {
  performanceReviewId: number;
  fromEmployeeId: number;
  type: FeedbackType;
  comments: string;
}
